SampleApp/Sample9.cs

```diff
@@ -51,8 +51,19 @@ public static class Sample9
         /// </summary>
         /// <param name="outputDir"></param>
         /// <returns></returns>
-        public static string RunSample9(DirectoryInfo outputDir)
+        public static string RunSample9()
         {
+            Stream stream = File.Open(@"C:\Users\Stenrud\Desktop\Book1.xlsx", FileMode.Open, FileAccess.Read, FileShare.ReadWrite);

```

Comment:
Can we get rid of the local file ref?

Reply:
Yes, of course. The changes to that file weren't suppose to be committed